src/components/leaderboard/ducks/actions.ts

```diff
@@ -0,0 +1,39 @@
+import { genericAsyncActions } from '../../../utils/asyncRequest';
```

Comment:
Why is this in the components file? Shouldn't it be in `user-leaderboard` and `team-leaderboard` containers?

Reply:
So that we only need one set of actions for both leaderboards instead of having 2 identical action files in each container.

Reply:
I think it makes more sense to have two different action files, that way if they diverge then logically we don't have to worry about rewriting.